.rubocop.yml

```diff
@@ -1,5 +1,3 @@
-require: rubocop-rspec
```

Comment:
I'll merge this, though I think we will want to add this back once Rubocop 0.38.1 is released.